garb/garb_config.cpp

```diff
@@ -46,12 +46,14 @@ Config::Config (int argc, char* argv[])
       cfg_     (),
       recv_script_ (),
       workdir_ (),
+      debug_   (false),
       exit_    (false)
 {
     po::options_description other ("Other options");
     other.add_options()
         ("version,v", "Print version & exit")
         ("help,h",    "Show help message & exit")
+        ("debug",     "Enable debug prints")
```

Comment:
Would it be a big effort if we allow it in the config file as well?

Reply:
Done. The only limitation with allowing the variable to be allowed in the config file is that we have to use it like `--debug=1` instead of just `--debug` when passing from command line.